src/memgraph.cpp

```diff
@@ -472,6 +478,22 @@ struct SessionData {
 DEFINE_string(auth_user_or_role_name_regex, memgraph::glue::kDefaultUserRoleRegex.data(),
               "Set to the regular expression that each user or role name must fulfill.");
 
+void InitFromCypherlFile(memgraph::query::InterpreterContext &ctx, std::string cypherl_file_path) {
+  memgraph::query::Interpreter interpreter(&ctx);
+  std::ifstream file(cypherl_file_path);
+  if (file.is_open()) {
+    std::string line;
+    while (std::getline(file, line)) {
+      if (!line.empty()) {
+        auto results = interpreter.Prepare(line, {}, {});
```

Comment:
Does this work with auth stuff, since you are not passing the username in the query?

Reply:
While you don't log in you can do whatever you want, so with this, every user created will have all privileges you then manage by either denying some or revoking all and then granting some.

Reply:
But what about your second PR, if you create a user first, how can you load all the data inside without authenticating, or will the load always happen before user creation?

Reply:
You can do whatever you want since I'm already in the database and  I'm not connecting I don't need to authenticate. If you set init_file it will delete all other users and then start filling database with users from file.

Reply:
I don't think that init_file should delete any users at all, I think it makes sense to run init_file first and then create a user from env, variables or file if one is specified, otherwise, you cannot pass the username to the Interpreter.
But besides that, there is another issue Imagine the following use case.
Memgraph was started with `init_file` and it was populated in the start and was running for a while, and then it crashed. When you start it a second time would you expect that the `init_file` removes all previous data and auth settings, or that we can detect that initialization already happened and we don't want another import? I think the `init_file` is meant to be run only once, even if the flag persists, we should not run the init file twice. What do you think?

Reply:
I discussed with team and I agree with you.